hack/patch-istio-crd.sh

```diff
@@ -0,0 +1,288 @@
+#!/bin/bash
```

Comment:
I really don't link how this is written in shell. It's complex to read and fragile I guess.

:(

Reply:
What do you mean by complex to read and fragile?